altair/api.py

```diff
@@ -233,6 +293,18 @@ def _shape_changed(self, name, old, new):
             self.shape._infer_type(self.parent.data)
 
 
+class Filter():
```

Comment:
Great, many thanks that is helpful. I will start to look through the
different milestones and issues and provide comments. I would imagine that
we will try to follow the spec as it evolves for now.

One question - we expect that Altair users are going to *quickly* find it
frustrating that the different python renderers (matplotlib, bokeh, plotly)
can handle layering and composition while the vega-lite/vega can't. There
are a number of extremely simple plot types (scatter + regression line)
that can't easily be rendered without that.

I am concerned that this will put pressure on us (Altair) to prematurely
fork/extend the spec informally to enable these things in the python based
renderers. Can you comments on the time frame for basic layering (multiple
sets of marks on the same axes)? I understand (and support) your desire to
evolve the spec in a careful and consistent manner, but I also worry about
the different python libraries starting to route around its limitations.

On Wed, Oct 14, 2015 at 11:25 PM, Kanit Wongsuphasawat <
notifications@github.com> wrote:

> In altair/api.py
> <https://github.com/ellisonbg/altair/pull/68#discussion_r42080187>:
>
> > @@ -233,6 +293,18 @@ def _shape_changed(self, name, old, new):
> >              self.shape._infer_type(self.parent.data)
> >
> >
> > +class Filter():
>
> We plan to release 0.8 (producing working vega 2 specs) some times next
> week (before VIS2015 <http://ieeevis.org/>). Then shortly after, we will
> try to refactor and freeze our syntax in 0.9 to reduce the churn for you
> guys. This might happen before VIS2015 but more likely shortly after the
> conference.
>
> Now, we also put our roadmap here:
> https://github.com/vega/vega-lite/wiki/Roadmap
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ellisonbg/altair/pull/68/files#r42080187>.
>



-- 
Brian E. Granger
Associate Professor of Physics and Data Science
Cal Poly State University, San Luis Obispo
@ellisonbg on Twitter and GitHub
bgranger@calpoly.edu and ellisonbg@gmail.com